src/codegeneration/TemplateLiteralTransformer.js

```diff
@@ -198,6 +199,21 @@ export class TemplateLiteralTransformer extends TempVarTransformer {
     return ParseTreeTransformer.prototype.transformFunctionBody.call(this, tree);
   }
 
+  transformNewExpression(tree) {
+    if (tree.operand) {
+      var operand = tree.operand;
+      do {
+        // If operand contains a tagged TemplateLiteral, parenthesize the entire
+        // operand.
+        if (operand.operand && operand.type === TEMPLATE_LITERAL_EXPRESSION) {
+          tree.operand = new ParenExpression(null, tree.operand);
+          break;
+        }
+      } while (operand = operand.operand);
+    }
+    return ParseTreeTransformer.prototype.transformNewExpression.call(this, tree);
```

Comment:
Use `super.transformNewExpression` (I see that there is some old code above that uses ES5 style super)